packages/inter-protocol/src/price/roundsManager.js

```diff
@@ -26,7 +26,7 @@ const V3_NO_DATA_ERROR = 'No data present';
 /** @type {bigint} */
 export const ROUND_MAX = BigInt(2 ** 32 - 1);
 
-const trace = makeTracer('RoundsM', false);
+const trace = makeTracer('RoundsM', true);
```

Comment:
Do you mean to turn that on in production?
(I don't object. I'm just checking that you didn't mean to turn it back off.)

Reply:
I went back and forth. I settled on yes, because we're diagnosing issues here, and it would give some added visibility.